client/extensions/woocommerce/woocommerce-services/views/packages/add-package-presets.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SelectOptGroups from 'components/forms/select-opt-groups';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+
+const defaultPackages = {
+	label: 'Custom box',
+	options: [
+		{
+			value: 'default_box',
+			label: 'Box',
+			is_user_defined: true,
+		},
+		{
+			value: 'default_envelope',
+			label: 'Envelope',
+			is_user_defined: true,
+		},
+	],
+};
+
+const getOptionGroups = ( presets ) => {
+	const groups = [ defaultPackages ];
+	if ( presets ) {
+		groups.push( {
+			label: presets.title,
+			options: presets.boxes.map( ( box, idx ) => {
+				return {
+					value: 'preset_' + idx,
+					label: box.name,
+				};
+			} ),
+		} );
+	}
+
+	return groups;
+};
+
+const handleSelectEvent = ( e, selectDefault, selectPreset, setSelectedPreset ) => {
+	const parts = e.target.value.split( '_' );
+	const type = parts[ 0 ];
+	const id = parts[ 1 ];
+	setSelectedPreset( e.target.value );
+	switch ( type ) {
+		case 'default':
+			return selectDefault( id );
+		case 'preset':
+			return selectPreset( Number.parseInt( id ) );
+	}
+};
+
+const AddPackagePresets = ( { siteId, selectedPreset, setSelectedPreset, presets, setModalErrors, updatePackagesField, translate } ) => {
+	const onSelectPreset = ( idx ) => {
+		const preset = presets.boxes[ idx ];
+		setModalErrors( siteId, {} );
+		updatePackagesField( siteId, {
+			index: null,
+			is_user_defined: false,
+			...preset,
+		} );
+	};
+
+	const onSelectDefault = ( value ) => {
+		setModalErrors( siteId, {} );
+		updatePackagesField( siteId, {
+			index: null,
+			is_letter: 'envelope' === value,
+			name: null,
+			is_user_defined: true,
+			outer_dimensions: null,
+			inner_dimensions: null,
+			box_weight: null,
+			max_weight: null,
+		} );
+	};
+
+	const onChange = ( event ) => handleSelectEvent( event, onSelectDefault, onSelectPreset, setSelectedPreset.bind( null, siteId ) );
```

Comment:
Instead of an explicit bind, how about using the more up-to-date way `=>` like you did here instead  https://github.com/Automattic/wp-calypso/pull/16540/files#diff-f86189d728a2fd78b98daa284ab2da14R149